cmd/gitbase/command/server.go

```diff
@@ -39,9 +43,10 @@ type Server struct {
 	PilosaURL     string   `long:"pilosa" default:"http://localhost:10101" description:"URL to your pilosa server" env:"PILOSA_ENDPOINT"`
 	IndexDir      string   `short:"i" long:"index" default:"/var/lib/gitbase/index" description:"Directory where the gitbase indexes information will be persisted." env:"GITBASE_INDEX_DIR"`
 	DisableSquash bool     `long:"no-squash" description:"Disables the table squashing."`
-	// IgnoreGitErrors by default when gitbase encounters and error in a
-	// repository it stops the query. With this parameter it won't complain and
-	// just skip those rows or repositories.
+	TraceEnabled  bool     `long:"trace" env:"GITBASE_TRACE" description:"Enables jaeger tracing"`
+	TraceEndpoint string   `long:"trace-endpoint" env:"GITBASE_TRACE_ENDPOINT" default:"localhost:6831" description:"Sets endpoint to send traces"`
```

Comment:
There are some jaeger standard env vars that we can use instead of this:https://github.com/jaegertracing/jaeger-client-java/tree/master/jaeger-core#configuration-via-environment